apps/participants/serializers.py

```diff
@@ -49,6 +49,22 @@ def save(self):
                                                  team=self.participant_team)
 
 
+class ParticipantSerializer(serializers.ModelSerializer):
+
+    member_name = serializers.SerializerMethodField()
+    member_id = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Participant
+        fields = ('member_id', 'member_name')
```

Comment:
@deshraj 
If we include all the fields in Participant model by:
fields=('user','status'.'team')
then serializer returns just id of user and team as output